theme/sass/components/_accordion-component.scss

```diff
@@ -0,0 +1,58 @@
+.accordion-component {
+
+  .accordion {
+    background-color: $accent-color-one;
+    color: $secondary-color;
+    font-weight: bold;
+    cursor: pointer;
+    padding: 18px;
+    text-align: left;
+    border: none;
+    outline: none;
+    transition: 0.4s;
+    width: 100%;
+    border-radius: 10px;
+    display: flex;
+
+    h3 {
+      display: flex;
+      flex: 1 1 0%;
+    }
+
+    .icons {
+      display: flex;
```

Comment:
If you add `padding: 15px;` here it will centralise the icons.

Reply:
done